package/execd/execd.conf

```diff
@@ -1,2 +1,4 @@
+task log:prio:local1.notice,tag:container-flush <pid/syslogd> \
+	[S] container flush -- Cleaning up lingering containers
 service log:prio:local1.err,tag:container \
-	[2345] execd /run/containers/queue /var/lib/containers/active -- Container job runner
+	[2345] execd -l debug /run/containers/queue -- Container job runner
```

Comment:
is the -l debug intended?

Reply:
Oups, nice catch! Was definitely not intended for prod :sweat_smile: